create_acme.yml

```diff
@@ -1,7 +1,9 @@
 ---
-# This playbook creates the ACME controler in a project.
-# The definition files are lookups targeting openshift-acme GitHub repository.
-# https://github.com/tnozicka/openshift-acme
+# This playbook creates a cert-manager ACME Issuer a namespace.
```

Comment:
```suggestion
# This playbook creates a cert-manager ACME Issuer in a namespace.
```

Reply:
Thanks! I fixed id, I meant "for a namespace".